job/storage/postgres/postgres.go

```diff
@@ -0,0 +1,108 @@
+package postgres
+
+import (
+	"database/sql"
+	"encoding/json"
+	"fmt"
+
+	_ "github.com/lib/pq"
+
+	"github.com/ajvb/kala/job"
+
+	log "github.com/Sirupsen/logrus"
+)
+
+var (
+	table = "jobs"
+)
+
+type DB struct {
+	conn *sql.DB
+}
+
+// New instantiates a new DB.
+func New(dsn string) *DB {
+	connection, err := sql.Open("postgres", dsn)
+	if err != nil {
+		log.Fatal(err)
+	}
+	// passive attempt to create table
+	_, _ = connection.Query(fmt.Sprintf(`create table %s (job jsonb);`, table))
```

Comment:
`database/sql` has built in parameterization (https://golang.org/pkg/database/sql/#DB.Query), can we use that instead of `Sprintf`?

Reply:
unfortunately dynamic table names falls outside of the realms of parameterization :(